rules/windows/discovery_process_discovery_via_tasklist_command.toml

```diff
@@ -15,31 +15,31 @@ false_positives = [
 ]
 from = "now-9m"
 index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
-language = "kuery"
+language = "eql"
 license = "Elastic License v2"
 name = "Process Discovery via Tasklist"
```

Comment:
super noisy, not sure if a user will be interested to be alerted for every tasklist execution

Reply:
agree 💯, i have no idea when/why this was created

Reply:
++

Reply:
let's separately deprecate it, feel free to keep the changes as-is in this PR

Reply:
these process start only rules were inherited and have been on our list to deprecate for quite some time.